### PR TITLE
Fix to support Django 4.0

### PR DESCRIPTION
**Pull request by kegan**
Update project to support Django 4.0. Minimum support is now Django 3.0.

Currently, the project is broken for Django 4.0 due to the following:
- removal of `from django.utils.encoding import force_text`
- removal of `from django.utils.translation import ugettext_lazy as _`

This is highlighted in issue https://github.com/svleeuwen/sortedm2m-filter-horizontal-widget/issues/16.

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from django.db import models
```
Comment (svleeuwen) [inline models.py]: Hi @kegan, looks like you added this `models.py` and `admin.py` to the root. Don't think these should be here, can you remove them?

Comment (svleeuwen) [reply models.py]: Hi @kegan, would you be able to look into the above?

Comment (kegan) [reply models.py]: @svleeuwen My apology for the super late reply 😬 I have fixed it by removing both `models.py` and `admin.py` from the commits.

Hope you can check if this is ok.

**Comment by compmonks** [comment]
Hello, is there any chance this merge happen anytime soon?
Since it fixes compatibility with django 4.0 it would certainly be useful to release a new package as well :)

**Pull request merged**